config/default.json

```diff
@@ -4,6 +4,7 @@
   "anchorControllerEnabled": false,
   "cronExpression": "0/5 * * * ? *",
   "expirationPeriod": 5400000,
+  "merkleDepthLimit": 0,
```

Comment:
I assume we will change this to a reasonable default once we figure out what a reasonable default would be.